src/ui/components/pages/LangsSettings.module.css

```diff
@@ -0,0 +1,77 @@
+.content {
+  width: 100%;
+  position: relative;
+  padding: 64px 60px 0 60px;
+  overflow: auto;
+  height: calc(100% - 48px);
+}
+
+.lang {
+  border-bottom: 1px solid #e9e9eb;
+  display: flex;
+  height: 44px;
+  position: relative;
+  width: 100%;
+}
+
+.lang.selected div {
+  color: #999999;
+}
+
+.langsList {
+  padding-bottom: 60px;
+}
+
+.flagIcon {
+  height: 44px;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 18px;
+}
+
+.selectButton {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  font-weight: 400;
+  height: 44px;
+}
+
+.langsConfirm {
+  background-color: #ffffff;
+  bottom: 0;
+  left: 0;
+  padding: 20px 0;
+  position: sticky;
+  right: 0;
+}
+
+.langsConfirm::before {
+  background: -moz-linear-gradient(
+    top,
+    rgba(255, 255, 255, 0) 0%,
+    rgba(255, 255, 255, 1) 61%,
+    rgba(255, 255, 255, 1) 100%
+  );
+  background: -webkit-linear-gradient(
+    top,
+    rgba(255, 255, 255, 0) 0%,
+    rgba(255, 255, 255, 1) 61%,
+    rgba(255, 255, 255, 1) 100%
+  );
```

Comment:
These prefixed versions are not needed for a long time already: https://caniuse.com/css-gradients